libraries/botbuilder-dialogs-adaptive/src/templates/staticActivityTemplate.ts

```diff
@@ -33,10 +34,18 @@ export class StaticActivityTemplate
      */
     public activity: Partial<Activity>;
 
+    /**
+     * @param _property The key of the conditional selector configuration.
+     * @returns The converter for the selector configuration.
+     */
     public getConverter(_property: keyof StaticActivityTemplateConfiguration): Converter | ConverterFactory {
         return undefined;
     }
 
+    /**
+     * @param config The configuration.
+     * @returns A object with the given configuration.
+     */
```

Comment:
For this case i think is valid that doesnt have a description, it gets inherited from the Configurable abstract class, we should do the same for the getConverter method